### PR TITLE
For review 

### PR DESCRIPTION
**Pull request by dburtnja**

### CODE REVIEW

File: exercise_0/main.cpp [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
     else if (is_vowels(character))
-        (*vowels)++;
+        vowels++;
     else
-        (*consonants)++;
+        consonants++;
 }
 
-int     words_checker(std::istream &read_file, char const *stop) {
+int     words_checker(std::istream &read_file, const std::string &stop = {}) {
```
Comment (reczepio) [inline exercise_0/main.cpp]: "words_checker" is good name for class, not for function. Widely spread convention said:
-function name should be "from verb"
-class name should be "from noun"

File: exercise_0/main.cpp [hunk 2]
```diff
@@ -36,9 +33,9 @@ int     words_checker(std::istream &read_file, char const *stop) {
     consonants = 0;
```
Comment (reczepio) [inline exercise_0/main.cpp:33]: Shorter:
int vowels = 0;
int consonants = 0;

File: exercise_0/main.cpp [hunk 3]
```diff
@@ -36,9 +33,9 @@ int     words_checker(std::istream &read_file, char const *stop) {
...
     others = 0;
```
Comment (reczepio) [inline exercise_0/main.cpp:34]: int others =0

File: exercise_0/main.cpp (hunk 2 above)
Comment (dburtnja) [reply exercise_0/main.cpp:33]: Yes, I know. It's bad style to write like that?

Comment (reczepio) [reply exercise_0/main.cpp:33]: No, it's not "bad". But, as long as we do not have pay for each line of code, I do not see why write it in this way.

Comment (dburtnja) [reply exercise_0/main.cpp:33]: Maybe someday:) For me, it's just more readable.

Comment (reczepio) [reply exercise_0/main.cpp:33]: If so, I'm OK with it.

File: exercise_0/main.cpp [hunk 4]
```diff
@@ -59,6 +59,7 @@ int     read_from_file(char *file_name) {
         return 1;
     }
     words_checker(input_file, nullptr);
+    input_file.close();
```
Comment (reczepio) [inline exercise_0/main.cpp]: Not needed. Please see "RAII" patern

Comment (reczepio) [reply exercise_0/main.cpp]: I cannot add comment directly to line 56.
Please open file in constructor - again, RAII pattern.

Comment (dburtnja) [reply exercise_0/main.cpp]: Fix is present in **9ed9ba**2500aa69d54bf65f5b83708fc3966015ff commit.

**Review by pstanisz — APPROVED** [review]

**Pull request merged**